solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrClientFactory.java

```diff
@@ -57,13 +57,18 @@ public CloudSolrClient createCloudSolrClient(String zookeeperConnectionString) {
 
     ConnectStringParser parser = new ConnectStringParser(zookeeperConnectionString);
 
-    CloudSolrClient.Builder cloudBuilder = new CloudSolrClient.Builder(
-        parser.getServerAddresses().stream()
-            .map(address -> String.format(Locale.ROOT, "%s:%s", address.getHostString(), address.getPort()))
-            .collect(Collectors.toList()),
-        Optional.ofNullable(parser.getChrootPath()));
-
-    cloudBuilder.withConnectionTimeout(settings.getHttpConnectionTimeout())
+    CloudSolrClient.Builder cloudBuilder =
+        new CloudSolrClient.Builder(
+            parser.getServerAddresses().stream()
+                .map(
+                    address ->
+                        String.format(
```

Comment:
Simple string concatenation with a colon would be simpler and shorter, probably wouldn't wrap

Reply:
Fixed in 6f59d6fa54dd1711174bd1e607b7c14e54fa6cd7